atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CQLKeyValueServices.java

```diff
@@ -59,6 +59,7 @@
 
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 
+@SuppressFBWarnings("SLF4J_ILLEGAL_PASSED_CLASS")
```

Comment:
The logger in CQLKeyValueService**_s_** is for CQLKeyValueService.class.  Is there a good reason for this?